src/components/Question.js

```diff
@@ -17,6 +17,7 @@ const Question = ({ question, handleAnswerSelect }) => {
                 value={answer.id}
                 onClick={handleAnswerSelect}
                 label={`${answer.text}`}
+                defaultChecked={answer.id === selectedAnswer}
```

Comment:
Although `defaultChecked` works here, technically it should be `checked`. `defaultChecked` means "this is what should be checked by default before the user makes an input" which is not what is happening here

Reply:
Done